### PR TITLE
Update billing policy

### PR DESCRIPTION
**Pull request by Annyv2**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/policies/billing.md [hunk 1]
```diff
@@ -8,5 +8,4 @@ The following policy governs requests for billing mechanisms within the Auth0 da
 
 ## Change of billing email address
 
-It is possible to change the billing email address, but for security reasons the request must be done from a Dashboard administrator. 
-The request should be performed by a Dashboard administrator. It should file a new ticket in our Support Center. A member of our support team will contact you in that same ticket with steps on how this request will proceed.
+It is possible to change the billing email address, but for security reasons the request should be performed by a Dashboard administrator by filing a new ticket trough our [Support Center](https://support.auth0.com). A member of our support team will then contact you in that same ticket with steps on how the request will proceed.
```
Comment (jeffreylees) [inline articles/policies/billing.md]: Typo "trough" -> "through"

**Comment by ntotten** [comment]
@Annyv2 Seems there is a conflict with changes @lobo made.

**Comment by lobo** [comment]
I'll create another PR, no worries. Thanks!

**Pull request closed without merging**